src/System.Reflection.Emit.ILGeneration/tests/ILGenerator/Emit4Tests.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class ILGeneratorEmit4
+    {
+        [Theory]
+        [InlineData(1, 1, 2)]
```

Comment:
Nit: It is a bit overkill to make this theory since it is used just for a single variant, and it is unlikely we will need more variants of the input in future.